DualSupportClientSample/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Microsoft.VisualStudio.Services.Client;
+using Microsoft.VisualStudio.Services.WebApi;
+using Microsoft.TeamFoundation.Core.WebApi;
+using System.Collections.Generic;
+
+namespace DualSupportClientSample
+{
+    //After running the console will close so please add a breakpoint or sleep to see output.
+    class Program
+    {
+        //============= Config [Edit these with your settings] =====================
+        internal const string vstsOrTfsCollectionUrl = "VSTS: https://*.visualstudio.com or TFS: https://*:8080/tfs/defaultcollection"; //change to the URL of your VSTS or TFS account 
```

Comment:
I suggest changing this to match the previous examples.  Having the comment inside the quotes is confusing.